gem_tasks/cucumber.rake

```diff
@@ -0,0 +1,27 @@
+# Copied from https://github.com/cucumber/cucumber-ruby/blob/master/gem_tasks/cucumber.rake
+
+require 'cucumber/rake/task'
+require 'cucumber/platform'
+
+class Cucumber::Rake::Task
+  def set_profile_for_current_ruby
```

Comment:
The cost is to forget about updating this file.
Is there an automatic way of doing that? Shouldn't cucumber take care of this for us?